compute_tools/src/bin/compute_ctl.rs

```diff
@@ -265,4 +268,18 @@ mod test {
     fn verify_cli() {
         Cli::command().debug_assert()
     }
+
+    #[test]
+    fn parse_pg_ext_gateway_address() {
+        let arg = "http://pg-ext-s3-gateway2";
+        let result = super::parse_remote_ext_config(arg).unwrap();
+        assert_eq!(result, arg);
+
+        let arg = "pg-ext-s3-gateway";
+        let result = super::parse_remote_ext_config(arg).unwrap();
+        assert_eq!(
+            result,
+            "http://pg-ext-s3-gateway.pg-ext-s3-gateway.svc.cluster.local"
```

Comment:
Should you use your constant here as well?

Reply:
Yes, but for that I'd need to move the `const` outside of the function, which unnecessarily pollutes the scope with a name which is only used for one single function just once.

Reply:
Let's see if someone else suggests this :-) I am not against, just prefer not to pollute the scope when possible, leaving everthing as private it can be. I wasn't even sure about this unit test, as it just covers one if/else branch :-) It was mainly for me to confirm it worked locally, but I decided to let it be, so that in the future we can keep track of the changes within this function parser implementation.